apps/style/common.scss

```diff
@@ -435,9 +443,17 @@ button.arrow:disabled {
   #codeWorkspace {
     left: $width;
   }
+  // specificially do not(.applab) so that we override the equivalent rule
+  // defined outside the media query
+  #codeWorkspace:not(.applab) {
```

Comment:
I've never seen anyone use the `:not` selector... does that make this a code smell?  Either way, I imagine this is a work-in-progress affair.